<ui version="4.0" >
 <class>BenchmarkWindow</class>
 <widget class="QWidget" name="BenchmarkWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Benchmark KDChart </string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QScrollBar" name="hSBar" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>664</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>999</number>
   </property>
   <property name="singleStep" >
    <number>10</number>
   </property>
   <property name="pageStep" >
    <number>100</number>
   </property>
   <property name="value" >
    <number>500</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="chartFrame" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>681</width>
     <height>647</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>300</width>
     <height>400</height>
    </size>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="vSBar" >
   <property name="geometry" >
    <rect>
     <x>729</x>
     <y>9</y>
     <width>16</width>
     <height>647</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="maximum" >
    <number>999</number>
   </property>
   <property name="singleStep" >
    <number>10</number>
   </property>
   <property name="pageStep" >
    <number>100</number>
   </property>
   <property name="value" >
    <number>500</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="BarChart" />
 </resources>
 <connections/>
</ui>
